src/components/core/menu/mainMenu/MainMenuRight.tsx

```diff
@@ -18,11 +20,13 @@ const TenderlyForkAlert = () => {
 export const MainMenuRight: FC = () => {
   const { menuMapping } = useBurgerMenuItems();
   const { aboveBreakpoint } = useBreakpoints();
+  const { networks } = useNetworkMenu();
 
   return (
-    <div className="flex items-center space-x-20">
+    <div className="flex items-center space-x-10 sm:space-x-20">
```

Comment:
use gap instead of space-x. Gap is better than margin because it applies on the parent instead of this children & doesn't collapse.

Reply:
Using gap now